requirements/dev.in

```diff
@@ -1,62 +1,26 @@
 # Dependencies needed only for development/testing.
-colorama==0.4.3
-coverage==5.0.3
-pytest-cov==2.8.1
-pytest==5.3.5
+pytest-cov
 django-debug-toolbar==2.2
```

Comment:
What's the reason to leave some pinned and some unpinned?

Reply:
The pinning here is to ensure the django version is identical from `dev.in` and `common.in`, and speed install; installing only one django instead of two.

**If** django can be unpinned, then this can be unpinned, but that is an experiment for another PR.

Reply:
Got it. Whatever you choose.